README.md

```diff
@@ -162,7 +162,7 @@ You can also [build the app manually][BUILD].
 
 ## Run
 
-Plug an Android device, and execute:
+Plug in an Android device to your computer, and execute:
```

Comment:
Hi, sorry for butting in, but I think "Plug an Android device into your computer" would be better.